app/views/errors/internal_server_error_500.html.erb

```diff
@@ -0,0 +1,29 @@
+<% provide :head_tags do %>
+  <%= javascript_tag do %>
+    <%= render partial: 'page_not_found_404', formats: [:js] %>
+  <% end %>
+<% end %>
+
+<div class="grid-container">
+  <h1 class="usa-prose-h1 margin-top-128">Internal server error</h1>
+  <p class="grid-col-9 usa-prose-intro margin-bottom-105">
+    We're sorry, something went wrong.  We're working to fix it as soon as we can.
+    Please check back in 30 minutes.
+  </p>
+
+  <div class="grid-col-7 usa-prose-body margin-bottom-205">
+    <p class="margin-bottom-105">
+      Send an email to <a class="internal-server-error-link" href="mailto:marketplace@va.gov">marketplace@va.gov</a> (or click the button below) to tell us
```

Comment:
we can use `usa-link` here for the class

Reply:
Got it.